content/applications/general/voip/onsip.rst

```diff
@@ -5,94 +5,146 @@ Use VoIP services in Odoo with OnSIP
+   click on :guilabel:`Users` and then select the user. By default the user will open on the
+   :guilabel:`User Info` tab. Click on the :guilabel:`Phone Settings` tab to reveal OnSIP
+   configuration credentials (first column).
 
-- **SIP Login / Browser's Extension**: the OnSIP 'Username'
-- **OnSIP authorization User**: the OnSIP 'Auth Username'
-- **SIP Password**: the OnSIP 'SIP Password'
-- **Handset Extension**: the OnSIP 'Extension'
+Fill in the following fields with the associated credentials listed below:
 
-You can find all this information by logging in at https://admin.onsip.com/users, then select the
-user you want to configure and refer to the fields as pictured below.
+- :guilabel:`SIP Login` / :guilabel:`Browser's Extension` = OnSIP :guilabel:`Username`
+- :guilabel:`OnSIP authorization User` = OnSIP :guilabel:`Auth Username`
+- :guilabel:`SIP Password` = OnSIP :guilabel:`SIP Password`
+- :guilabel:`Handset Extension` = OnSIP :guilabel:`Ext.` (extension without the `x`)
 
-.. image:: onsip/onsip03.png
+.. image:: onsip/onsip-creds.png
    :align: center
+   :alt: OnSIP user credentials with username, auth username, SIP password, and extension
+         highlighted.
 
-You can now make phone calls by clicking the phone icon in the top right corner of Odoo (make sure
-you are logged in as a user properly configured in Odoo and in OnSIP).
+.. tip::
+   The OnSIP extension can be found in the :guilabel:`User` banner line above the tabs.
 
-.. note::
-   If you see a *Missing Parameters* message in the Odoo softphone, make sure to refresh your Odoo
-   window and try again.
+Once these steps are complete, and the work is saved, Odoo users can make phone calls by clicking
+:guilabel:`📞 (phone)` icon in the top-right corner of Odoo.
 
-   .. image:: onsip/onsip04.png
-      :align: center
+.. seealso::
+   Additional setup and troubleshooting steps can be found on `OnSIP's knowledgebase
+   <https://support.onsip.com/hc/en-us>`_.
+
+Incoming calls
+--------------
 
-.. note::
-   If you see an *Incorrect Number* message in the Odoo softphone, make sure to use the
-   international format, leading with the plus (+) sign followed by the international country code.
-   E.g.: `+16506913277` (where `+1` is the international prefix for the United States).
+The Odoo database will also receive incoming calls that produce pop-up windows in Odoo. Click the
+green :guilabel:`📞 (phone)` icon to answer the call.
 
-   .. image:: onsip/onsip05.png
+   .. image:: onsip/incoming-call.png
       :align: center
+      :alt: alt text
+
+.. seealso::
+   :doc:`voip_widget`
+
+Troubleshooting
+---------------
+
+Missing parameters
+~~~~~~~~~~~~~~~~~~
 
-You can now also receive phone calls. Your number is the one provided by OnSIP. Odoo will ring and
-display a notification.
+If a *Missing Parameters* message appears in the Odoo widget, make sure to refresh the Odoo
+browser window (or tab) and try again.
```

Comment:
I would double check line 112, because it looks like it is an early break (before 100 characters)